templates/form/textInput.tpl

```diff
@@ -43,7 +43,7 @@
 					{if $FBV_readonly} readonly="readonly"{/if}
 					value="{$FBV_value[$thisFormLocale]|escape}"
 					name="{$FBV_name|escape}[{$thisFormLocale|escape}]"
-					id="{$FBV_id|escape}-{$thisFormLocale|escape}{$uniqId}"
+					id="{$FBV_id|escape}-{$thisFormLocale|escape|escape:jqselector}{$uniqId}"
```

Comment:
Here's a case where jqselector is applied to $selector:javascript: 
https://github.com/kaschioudi/pkp-lib/blob/1911-fix/templates/linkAction/buttonGenericLinkAction.tpl#L19